neuro_config_client/factories.py

```diff
@@ -308,6 +308,7 @@ def _create_node_pool(self, payload: dict[str, Any]) -> NodePool:
             machine_type=payload.get("machine_type"),
             idle_size=payload.get("idle_size", NodePool.idle_size),
             is_preemptible=payload.get("is_preemptible", NodePool.is_preemptible),
+            zones=payload.get("zones", ()),
```

Comment:
`zones` info in NodePool is a legacy?
I see it in ClusterProvider.

Reply:
It's not legacy, it is used